index.html

```diff
@@ -50,6 +50,108 @@ <h5>LET'S CONNECT</h5>
   </div>
  </div>
 </section>
-</div>
+
+
+
+<Section class="Works">
+  <div class="work-container">
+                <div class="im"><img src="./images/Snapshoot Portfolio (1).svg" alt=""></div> 
+              <div class="caption-contaier">
+          <div><h2>Tonic</h2></div>
+           <div class="caption">
+               <p>CANOPY</p>
+          <div class="dot"></div>
+               <p>Back End Dev</p>
+          <div class="dot"></div>
+               <p>2015</p>
+          </div>
+            </div>
+      </div>
+    </div>
+           <p class="information">A daily selection of privately  personalized  reads; no accounts or sign-ups required.</p>
+     
+         <ul class="buttons">
+             <li><a href=""><img src="./images/Tag (3).svg" alt=""></a></li>
+             <li><a href=""><img src="./images/Tag (4).svg" alt=""></a></li>
+             <li><a href=""><img src="./images/Tag (5).svg"alt=""</a></li>
+         </ul>
+</Section>
```

Comment:
- [ ] Kindly make sure your design matches the original design 👍🏽 

Your Design | Figma Design
![screenshot (1)](https://user-images.githubusercontent.com/44624138/170845424-69765b1f-6d95-40bd-a7a5-e769d73a5348.png)
![screenshot](https://user-images.githubusercontent.com/44624138/170845426-e600eaa8-1608-45e1-a4cf-d7434017cb1c.png) | 
![screenshot (3)](https://user-images.githubusercontent.com/44624138/170845440-84c8c6c5-9662-4184-bb98-d93db92bb5ad.png)
![screenshot (4)](https://user-images.githubusercontent.com/44624138/170845441-984bc3ad-d9e9-4826-b2d6-6b9722808dab.png)

Reply:
Kindly populate the work section as seen on Figma for a better user interface 😉